google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/vcs/UploadSourceAction.java

```diff
@@ -176,20 +183,17 @@ public void run(@NotNull ProgressIndicator indicator) {
 
         GitRepositoryManager repositoryManager = GitUtil.getRepositoryManager(project);
         final GitRepository repository = repositoryManager.getRepositoryForRoot(root);
-        LOG.assertTrue(repository != null, "GitRepository is null for root " + root);
         if (repository == null) {
-          SwingUtilities.invokeLater(new Runnable() {
-            @Override
-            public void run() {
-              Messages.showErrorDialog(project, GctBundle.message("uploadtogcp.failedtocreategit"),
-                  "Google");
-            }
-          });
+          SwingUtilities.invokeLater(() ->
+              Messages.showErrorDialog(project,
+                  GctBundle.message("uploadtogcp.failedtocreategit"), "Google"));
```

Comment:
Bundleize Google and add something else, i.e., Google Cloud Source Repositories?